docs/content-server-api/pointerChanges/get.md

```diff
@@ -0,0 +1,76 @@
+# Get Pointer Changes
+
+List all deployment changes from the same pointer
+
+**URL** : `/pointerChanges`
+
+<!-- **URL Parameters** : `pk=[integer]` where `pk` is the ID of the Account on the server. -->
+
+**Method** : `GET`
+
+**Auth required** : NO
+
+**Permissions required** : NO
+
+**Query Parameters** :
+
+
+
+*DEPRECATED: PLEASE USE from*
+- fromLocalTimestamp
+  - Format: int
+  - Value: timestamp
+  - Example: fromLocalTimestamp=1606829553969
+  - Default value: NULL - no lower filter is set
+  - Description: Acts as a filter in the collection of deployments, this value is the minimum value of local timestamp that any deployment in the collection will have.
+
+*DEPRECATED: PLEASE USE to*
+- toLocalTimestamp
+  - Format: int
+  - Value: timestamp
+  - Example: toLocalTimestamp=1606829553969
+  - Default value: NULL - no upper filter is set
+  - Description: Acts as a filter in the collection of deployments, this value is the maximum value of local timestamp that any deployment in the collection will have.
+
+- from
+  - Format: int
+  - Value: timestamp
+  - Example: from=1606829553969
+  - Default value: NULL - no lower filter is set
+  - Description: Acts as a filter in the collection of deployments, this value is the minimum value of timestamp (of the field indicated by SortingField: localTimestamp is the default) that any deployment in the collection will have.
+
+- to
+  - Format: int
+  - Value: timestamp
+  - Example: to=1606829553969
+  - Default value: NULL - no upper filter is set
+  - Description: Acts as a filter in the collection of deployments, this value is the maximum value of timestamp (of the field indicated by SortingField: localTimestamp is the default) that any deployment in the collection will have.
+
+- lastId
+  - Format: string
+  - Value: EntityId
+  - Example: lastId=QmNknKv8MuKbfZ73z4QdUEsNbTd1ZAN1fSuwTFGiNGeCt5
+  - Default value: -
+  - Description: It is the last entity id that will be listed in the collection
```

Comment:
If `lastId` is set, then it won't be listed at all, it should be avoided. Right? Or am I getting something wrong?

Reply:
You mean that the users shouldn't use that param unless they are using our auto-generated pagination?